x/delegation/keeper/keeper_suite_test.go

```diff
@@ -19,22 +22,23 @@ func TestDelegationKeeper(t *testing.T) {
 	RunSpecs(t, fmt.Sprintf("x/%s Keeper Test Suite", types.ModuleName))
 }
 
-func PayoutRewards(s *i.KeeperTestSuite, staker string, amount uint64) {
+func PayoutRewards(s *i.KeeperTestSuite, staker string, coins sdk.Coins) {
 	fundingState, found := s.App().FundersKeeper.GetFundingState(s.Ctx(), 0)
 	Expect(found).To(BeTrue())
 
+	// TODO support all denominations once the funding module supports it
```

Comment:
OPTIONAL
maybe you will have an easier time not creating a funded pool everytime where you have to make sure that the coins you want to pay out to the delegators have to be always funded and instead directly mint the coins to the pool module address here. I think this should be possible since the pool module does not keep track of how much assets it gets from the funders module and instead instantly moves the amount which it gets from ChargeFundersOfPool to the next module or account addresses.

Then you could also already test distributing rewards in multiple coins and don't have to wait for the funders module

Reply:
Agreed, thats even how I did it to test the multiple coins stuff. However, would leave this to a separate PR, then